lib/stack_master/stack_differ.rb

```diff
@@ -40,6 +50,16 @@ def output_diff
       end
     end
 
+    def noecho_keys
+      if @current_stack
+        @current_stack.parameters_with_defaults.select do |key, value|
```

Comment:
If AWS has blanked out the value with **** then it's assumed to be a noecho param.  Obviously this will fail someone uses an actual value of **** but that seems unlikely.